kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ModalContainer.kt

```diff
@@ -46,11 +46,13 @@ import com.squareup.workflow.ui.ModalContainer.Companion.forContainerScreen
  *
  * @param ModalRenderingT the type of the nested renderings to be shown in a dialog window.
  */
-abstract class ModalContainer<ModalRenderingT : Any>
-@JvmOverloads constructor(
+abstract class ModalContainer<ModalRenderingT : Any> @JvmOverloads constructor(
```

Comment:
Update commit message to note these changes to make the container views friendly to the layout editor.